CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## x.x.x
+* Added `CountlyFeedbacks` interface to manage feedback widgets, including NPS, Surveys, and Ratings.
+    * `presentNPS`, `presentSurvey`, and `presentRating` methods now allow displaying specific widgets by name, ID, or tag.
```

Comment:
it is great but mb we could add function signatures with class only not with parameter names like
presentNPS(String)

Reply:
i understand why @arifBurakDemiray is suggesting that but current version also feels totally fine to me